index.html

```diff
@@ -0,0 +1,149 @@
+        <a href="#" class="final_msj">LET'S CONNECT</a>
+        <ul class="header_list">
+            <li><a href="#" class="header_icon"><img src="./image/twitter_icon.png" alt="twitter_icon"></a></li>
+            <li><a href="#" class="header_icon"><img src="./image/Linkedin icon.png" alt="linkedin_icon"></a></li>
+            <li><a href="#" class="header_icon"><img src="./image/m_icon.png" alt="m_icon"></a></li>
+            <li><a href="#" class="header_icon"><img src="./image/github_icon.png" alt="github"></a></li>
+            <li><a href="#" class="header_icon"><img src="./image/V_icon.png" alt="V_icon"></a></li>
+        </ul>
+    </section>  
+    <section class="works">
+        <div class="works_card">
+            <img class="works_img" src="./image/works_1.png" alt="works_img">
+            <h3 class="works_title">Tonic</h3>
+            <ul class="works_options">
+                <li><a href="#" class="works_list1">CANOPY</a></li>
+                <li><a href="#" class="works_list">Back End Dev</a></li>
+                <li><a href="#" class="works_list">2015</a></li>           
+            </ul>
+            <p class="works_parragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="languages">
+                <p class="lang_skill">html</p>
+                <p class="lang_skill">css</p>
+                <p class="lang_skill">javaScript</p>
+            </div>
+            <a href="#" class="button">See Project</a>
+        </div>
+        <div class="works_card">
+            <img class="works_img" src="./image/works_2.png" alt="works_img">
+            <h3 class="works_title">Multi-Post<br>Stories</h3>
+            <ul class="works_options">
+                <li><a href="#" class="works_list1">CANOPY</a></li>
+                <li><a href="#" class="works_list">Back End Dev</a></li>
+                <li><a href="#" class="works_list">2015</a></li>           
+            </ul>
+            <p class="works_parragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="languages">
+                <p class="lang_skill">html</p>
+                <p class="lang_skill">css</p>
+                <p class="lang_skill">javaScript</p>
+            </div>
+            <a href="#" class="button">See Project</a>
+        </div>
+        <div class="works_card">
+            <img class="works_img" src="./image/works_3.png" alt="works_img">
+            <h3 class="works_title">Tonic</h3>
+            <ul class="works_options">
+                <li><a href="#" class="works_list1">CANOPY</a></li>
+                <li><a href="#" class="works_list">Back End Dev</a></li>
+                <li><a href="#" class="works_list">2015</a></li>           
+            </ul>
```

Comment:
- [ ] I would suggest you please stick as much as possible to the Figma design. You missed `counter` a small `.` please implement it. See the image below for reference
### Figma design
 
![figma](https://user-images.githubusercontent.com/81727988/192126288-d3258cd0-3504-4015-8ad2-a3d54127722f.png)
### You design
![your](https://user-images.githubusercontent.com/81727988/192126293-bcb5670e-553c-4448-ab35-a6d75e8d79f3.png)